components/favicon_title_handler/favicon_title_handler.tsx

```diff
@@ -83,21 +84,32 @@ class FaviconTitleHandler extends React.PureComponent<Props> {
             currentTeam,
             currentTeammate,
             unreads,
+            inGlobalThreads,
         } = this.props;
         const {formatMessage} = this.props.intl;
 
         const currentSiteName = siteName || '';
 
+        const mentionTitle = unreads.mentionCount > 0 ? '(' + unreads.mentionCount + ') ' : '';
+        const unreadTitle = !this.isDynamicFaviconSupported && unreads.messageCount > 0 ? '* ' : '';
+
         if (currentChannel && currentTeam && currentChannel.id) {
             let currentChannelName = currentChannel.display_name;
             if (currentChannel.type === Constants.DM_CHANNEL) {
                 if (currentTeammate != null) {
                     currentChannelName = currentTeammate.display_name;
                 }
             }
-            const mentionTitle = unreads.mentionCount > 0 ? '(' + unreads.mentionCount + ') ' : '';
-            const unreadTitle = !this.isDynamicFaviconSupported && unreads.messageCount > 0 ? '* ' : '';
-            document.title = mentionTitle + unreadTitle + currentChannelName + ' - ' + currentTeam.display_name + ' ' + currentSiteName;
+            document.title = `${mentionTitle + unreadTitle + currentChannelName} - ${currentTeam.display_name} ${currentSiteName}`;
+        } else if (currentTeam && inGlobalThreads) {
+            document.title = formatMessage({
+                id: 'globalThreads.title',
+                defaultMessage: '{prefix} Threads - {displayName} {siteName}',
```

Comment:
```suggestion
                defaultMessage: '{prefix}Threads - {displayName} {siteName}',
```
also, would it be possible to just `formatMessage` the word threads and use the same formula for both? it will prevent subtle errors like this to happen.

Reply:
I did consider that (to match above), but I felt it would be better to move towards what `sidebar.team_select` is like below.